src/js/menu/menu-button.js

```diff
@@ -298,19 +297,19 @@ class MenuButton extends Component {
   handleKeyDown(event) {
 
     // Escape or Tab unpress the 'button'
-    if (keycode.isEventKey(event, 'Esc') || keycode.isEventKey(event, 'Tab')) {
+    if (event.key === 'Esc' || event.key === 'Tab') {
```

Comment:
looks like during review, we missed that these should be `Escape` rather than `Esc`, which keycode likely helped with. Someone noticed this issue in slack.